diff --git a/toolchains/runtime-xxx/runtime-debugging/python/build.sh b/toolchains/runtime-xxx/runtime-debugging/python/build.sh
index 0ff8548..07872fa 100755
--- a/toolchains/runtime-xxx/runtime-debugging/python/build.sh
+++ b/toolchains/runtime-xxx/runtime-debugging/python/build.sh
@@ -1 +1 @@
-cc -o libpnic.so -I/usr/include/python3.11 -fPIC -shared pni.c -lpthread -lpython3.11
+cc -g -O0 -o libpnic.so -I/usr/include/python3.11 -fPIC -shared pni.c -lpthread -lpython3.11
diff --git a/toolchains/runtime-xxx/runtime-debugging/python/test.py b/toolchains/runtime-xxx/runtime-debugging/python/test.py
index d4240dd..3749eff 100755
--- a/toolchains/runtime-xxx/runtime-debugging/python/test.py
+++ b/toolchains/runtime-xxx/runtime-debugging/python/test.py
@@ -8,6 +8,7 @@ threading.current_thread()
 CPNIC_CB = CFUNCTYPE(c_int, c_char_p)
 
 def myFuncC(strparam):
+	import pdb; pdb.set_trace()
 	try:
 		print("callback string: %s" % strparam.decode("utf-8"))
 		return 0
